themes/default/layouts/page/automation.html

```diff
@@ -3,193 +3,422 @@
+                                            <p class="font-bold">Pulumipus’ Self-Service Platform, a web app in Python</p>
+                                            <p>
+                                                This is a self-service interface for creating customized static sites, databases,
+                                                virtual machines, and more. The code is open source and available on
+                                                <a href="https://github.com/komalali/self-service-platyform" target="_blank" rel="noreferrer noopener" class="link">
+                                                GitHub</a>.
+                                            </p>
+                                        </div>
+
+                                        <div class="lg:w-1/2 p-3">
+                                            <div class="text-center">{{ partial "color-icon.html" (dict "icon" "clipboard" "icon_color" "salmon") }}</div>
+                                            <p class="font-bold">Python IaC in your Jupyter notebook</p>
+                                            <p>
+                                                Use Pulumi’s Python SDK with Automation API to create infrastructure interactively
+                                                from a Jupyter notebook. Examples are available on
+                                                <a href="https://github.com/pulumi/automation-api-examples/tree/main/python/pulumi_via_jupyter" target="_blank" rel="noreferrer noopener" class="link">
+                                                GitHub</a>.
+                                            </p>
+                                        </div>
+                                    </div>
                                 </div>
                             </div>
-                            <div class="card-cta-btn">
-                                <a href="{{ relref . "/case-studies/snowflake" }}" class="btn-primary hidden md:inline-block">Learn How Snowflake Did It</a>
-                                <a href="{{ relref . "/case-studies/snowflake" }}" class="btn-primary md:hidden">Learn More</a>
+                            <div class="w-full lg:w-1/3 lg:order-first py-3 px-6 lg:p-3">
+                                <div class="card p-3 pb-8 relative bg-white">
+                                    <div class="w-2/3 mx-auto my-2 px-6">
+                                        <a href="{{ relref . "/case-studies/mercedes-benz" }}" class="link">
+                                            {{ partial "customer-logo.html" (dict "logo" "mercedes-benz") }}
+                                        </a>
+                                    </div>
+                                    <p>
+                                        Automation API powers custom infrastructure platforms for organizations like
```

Comment:
update the customer card for 'Self-Service Infrastructure':
<SANS Institute logo>
SANS Institute provides cybersecurity training and certification. It used Pulumi Automation API to build a self-service platform that enables instructors to provision virtual learning environments using an automated process.
[link to case study]

Reply:
👍